app/committees/controllers.py

```diff
@@ -123,6 +123,8 @@ def get_committee(committee_id, broadcast = False):
 @get_user
 def create_committee(user, user_data):
 
+    print(user_data)
```

Comment:
Do you still need this or was it just for testing? Should be removed if possible.

Reply:
Whoops. Removed